control-plane/connect-inject/container_init.go

```diff
@@ -16,6 +17,7 @@ const (
 	envoyUserAndGroupID         = 5995
 	copyContainerUserAndGroupID = 5996
 	netAdminCapability          = "NET_ADMIN"
+	dnsServiceHostEnvSuffix     = "DNS_SERVICE_HOST"
```

Comment:
What if there's a service like `foo-dns`? I think we need to use the Consul DNS service name exactly.

Reply:
Can we take in `-release-prefix` and convert to uppercase/`_`'s?

Reply:
✅